OpenSim/Simulation/SimbodyEngine/Body.h

```diff
@@ -36,14 +36,15 @@ class WrapObjectSet;
 //=============================================================================
 //=============================================================================
 /**
- * An OpenSim rigid body component. An OpenSim::Body is essentially a reference
- * frame with inertial properties specified by its mass, center-of-mass located
- * in the body frame and the moment of inertia tensor about the center-of-mass.  
+ * An OpenSim::Body is a PhysicalFrame (reference frame) with associated 
+ * inertia specified by its mass, center-of-mass located in the PhysicalFrame,
+ * and its moment of inertia tensor about the center-of-mass.
```

Comment:
I agree about the clarity. Since we're editing this anyway, can we change `center-of-mass located in the PhysicalFrame` to `center-of-mass specified in the PhysicalFrame`? Actually, I think we should remove the description of how these quantities are expressed in the properties for the class. That is specified in the property description anyway. I propose:

```
An OpenSim::Body is a PhysicalFrame (reference frame) with associated 
inertia specified by its mass, center of mass,
and its inertia tensor.
```